### PR TITLE
Update Rstudio Helm Chart

### PR DESCRIPTION
**Pull request by trispera**
- Add NGINX in front of Rstudio for authentication
- Route points NGINX
- Add secret with USER and PASSWORD to connect to Rstudio
- Update README and NOTES.txt

### CODE REVIEW

**Comment by lvarin** [comment]
Not working for me now

Screenshot from 2024-04-25 15-53-58

File: charts/rstudio/templates/NOTES.txt [hunk 1]
```diff
@@ -15,6 +15,11 @@ You can retrieve Rstudio URL via this command:
   export RSTUDIO=$(oc get route --namespace {{ .Release.Namespace }} -o yaml | yq '.items[0].spec.host')
```
Comment (lvarin) [inline charts/rstudio/templates/NOTES.txt:15]: missing route name

**Comment by trispera** [comment]
Did you wait for the build to complete before accessing it?

File: charts/rstudio/templates/NOTES.txt (hunk 1 above)
Comment (trispera) [reply charts/rstudio/templates/NOTES.txt:15]: Yes. It assumes you don't have other routes already deployed. With your namespace, it'll return other values.
Need to find a better way for this one. With annotations maybe 🤔

**Comment by lvarin** [comment]
Shame

No, i did not

**Comment by lvarin** [comment]
Now it works

File: charts/rstudio/templates/NOTES.txt (hunk 1 above)
Comment (lvarin) [reply charts/rstudio/templates/NOTES.txt:15]: You know the name of the route before hand, isn't it? Why not use that?

Comment (lvarin) [reply charts/rstudio/templates/NOTES.txt:15]:
```sh
$ oc get route rstudio -n test-helm-charts -o yaml | yq .spec.host
"pedropedro.2.rahtiapp.fi.2.rahtiapp.fi"
```

Comment (trispera) [reply charts/rstudio/templates/NOTES.txt:15]: With this update I know. It wasn't the case before. I'll do the change

Comment (trispera) [reply charts/rstudio/templates/NOTES.txt]: Pushed new commit

**Review by lvarin — APPROVED** [review]
LGTM

**Pull request merged**